libs/wire-api/src/Wire/API/MLS/Message.hs

```diff
@@ -318,22 +330,100 @@ instance SerialiseMLS (MessagePayload 'MLSPlainText) where
   -- so the next case is left as a stub
   serialiseMLS _ = pure ()
 
-newtype UnreachableUsers = UnreachableUsers {unreachableUsers :: [Qualified UserId]}
+newtype UnreachableUsers = UnreachableUsers {unreachableUsers :: NonEmpty (Qualified UserId)}
```

Comment:
You're right! I confused the `Semigroup a => Semigroup (Maybe a)` instance with `Alternative Maybe`. Removing the `<\>` operator and the manual definition of the Semigroup.

Reply:
Actually, I'll keep the manual instance of `Semigroup UnreachableUsers` because it drops the duplicates when joining two such lists. But `(<\>)` is going away for sure.